Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -216,48 +226,48 @@ function Stockpile_UpdateAction(furniture, deltaTime)
     end
 
 
-    -- TODO: In the future, stockpiles -- rather than being a bunch of individual
-    -- 1x1 tiles -- should manifest themselves as single, large objects.  This
-    -- would respresent our first and probably only VARIABLE sized "furniture" --
-    -- at what happenes if there's a "hole" in our stockpile because we have an
-    -- actual piece of furniture (like a cooking stating) installed in the middle
-    -- of our stockpile?
-    -- In any case, once we implement "mega stockpiles", then the job-creation system
-    -- could be a lot smarter, in that even if the stockpile has some stuff in it, it
-    -- can also still be requestion different object types in its job creation.
+	-- TODO: In the future, stockpiles -- rather than being a bunch of individual
+	-- 1x1 tiles -- should manifest themselves as single, large objects.  This
+	-- would respresent our first and probably only VARIABLE sized "furniture" --
+	-- at what happenes if there's a "hole" in our stockpile because we have an
+	-- actual piece of furniture (like a cooking stating) installed in the middle
+	-- of our stockpile?
+	-- In any case, once we implement "mega stockpiles", then the job-creation system
+	-- could be a lot smarter, in that even if the stockpile has some stuff in it, it
+	-- can also still be requestion different object types in its job creation.
 
     local itemsDesired = {}
 
-    if (furniture.Tile.Inventory == nil) then
-        --ModUtils.ULog("Creating job for new stack.")
-        itemsDesired = Stockpile_GetItemsFromFilter( furniture )
-    else
-        --ModUtils.ULog("Creating job for existing stack.")
-        desInv = furniture.Tile.Inventory.Clone()
-        desInv.MaxStackSize = desInv.MaxStackSize - desInv.StackSize
-        desInv.StackSize = 0
+	if( furniture.Tile.Inventory == nil ) then
+		--ModUtils.ULog("Creating job for new stack.")
+		itemsDesired = Stockpile_GetItemsFromFilter( furniture )
+	else
+		--ModUtils.ULog("Creating job for existing stack.")
+		desInv = furniture.Tile.Inventory.Clone()
+		desInv.MaxStackSize = desInv.MaxStackSize - desInv.StackSize
+		desInv.StackSize = 0
 
         itemsDesired = { desInv }
     end
 
-    local job = Job.__new(
-        furniture.Tile,
-        nil,
-        nil,
-        0,
-        itemsDesired,
-        Job.JobPriority.Low,
-        false
-    )
-    job.JobDescription = "job_stockpile_moving_desc"
-    job.acceptsAny = true
+	local j = Job.__new(
```

Comment:
Notice that I didn't change job to j. j was the original name, changed all of them to job, and lots of other things. Added an error somewhere in the file. And since we don't get Lua errors as we do with C#, it makes it harder to find and test. So I backed up the changes. I already changed several calls in Lua.